model/src/main/resources/META-INF/openapi.yaml

```diff
@@ -444,6 +373,139 @@ components:
                 additionalProp3: "zzz"
               signedOffBy: "signedOffByName <signedOffBy@example.com>"
 
+    multipleContentsResponse:
+      value:
+        contents:
+          - content:
+              type: ICEBERG_TABLE
+              id: "b874b5d5-f926-4eed-9be7-b2380d9810c0"
+              metadataLocation: "/path/to/metadata/"
+              snapshotId: 1
+              schemaId: 2
+              specId: 3
+              sortOrderId: 4
+            key:
+              elements:
+                - example
+                - key
+        effectiveReference:
+          type: BRANCH
+          hash: "abcdef4242424242424242424242beef00dead42112233445566778899001122"
+          name: "exampleBranch"
+
+    commitResponse:
+      value:
+        targetBranch:
+          type: BRANCH
+          hash: "abcdef4242424242424242424242beef00dead42112233445566778899001122"
+          name: "exampleBranch"
+        addedContents:
+          - contentId: "7fe924d9-45af-574e-9bbc-51b48077017e"
+            key:
+              elements:
+                - example
+                - key
+
+    contentResponseIceberg:
+      value:
+        content:
+          type: ICEBERG_TABLE
+          id: b874b5d5-f926-4eed-9be7-b2380d9810c0
+          metadataLocation: "/path/to/metadata/"
+          snapshotId: 1
+          schemaId: 2
+          specId: 3
+          sortOrderId: 4
+        effectiveReference:
+          type: BRANCH
+          hash: "abcdef4242424242424242424242beef00dead42112233445566778899001122"
+          name: "exampleBranch"
+
+    singleReferenceResponse:
+      value:
+        reference:
+          type: BRANCH
+          hash: "abcdef4242424242424242424242beef00dead42112233445566778899001122"
+          name: "exampleBranch"
+
+    singleReferencesResponseWithMetadata:
```

Comment:
typo? Should this be `singleReferenceResponseWithMetadata` (singlular "reference") as above?